### ISSUE TITLE
Pretrained models detect some classes but not all

### ISSUE THREAD

**Issue by mocialov** [facebookresearch/detectron2#1694]
I am looking for a pretrained model that segments highway road. By looking at detectrons model zoo, there are models pretrained on cityscapes dataset (https://github.com/facebookresearch/detectron2/blob/master/configs/Cityscapes/mask_rcnn_R_50_FPN.yaml) which has road class. However, when I run a pre-trained detectron model, it only gives me cars and people labels. 
Does that mean that the models in the model zoo are not pretrained on all the classes of the corresponding datasets?

**Comment by ppwwyyxx** [comment]
The model is an instance segmentation model. Cityscapes's instance classes do not include "road"
